dictionary.json

```diff
@@ -16966,6 +17623,20 @@
     "examples": [],
     "fields": []
   },
+  {
+    "toaq": "tuaja",
+    "type": "predicate",
+    "english": "▯ is brave/courageous.",
+    "gloss": "courageous",
+    "short": "",
+    "keywords": [],
+    "frame": "a",
+    "distribution": "d",
+    "pronominal_class": "ho",
+    "notes": [],
+    "examples": [],
+    "fields": []
+  },
   {
     "toaq": "tua",
     "type": "predicate",
```

Comment:
Below this: tuımy is now an invalid word

Reply:
#154